lib/stateParser.mly

```diff
@@ -193,6 +195,8 @@ atom_init:
 | atom { let x,v = $1 in x,(TyDef,v) }
 | typ=NAME loc=left_loc  { (loc, (Ty typ,ParsedConstant.zero)) }
 | ATOMIC typ=NAME loc=left_loc { loc,(Atomic typ,ParsedConstant.zero)}
+| loc=left_loc EQUAL TOK_NOP
+  { (loc,(Ty "string", mk_instr_val "NOP")) }
```

Comment:
The type of instructions in litmus is `ins_t` (with the appropriate `typedef` in litmus, here `Ty "ins_t"`). It would be nice to have the same type here, although this may implies other changes in other places...